openpype/hosts/blender/plugins/publish/extract_abc_model.py

```diff
@@ -0,0 +1,17 @@
+import pyblish.api
+from openpype.pipeline import publish
+
+
+class ExtractModelABC(publish.Extractor):
+    """Extract model as ABC."""
+
+    order = pyblish.api.ExtractorOrder - 0.1
+    label = "Extract Model ABC"
+    hosts = ["blender"]
+    families = ["model"]
+    optional = True
+
+    def process(self, instance):
+        # Add abc.export family to the instance, to allow the extraction
+        # as alembic of the asset.
+        instance.data["families"].append("abc.export")
```

Comment:
Why does this family have the `.export` suffix? What makes it so different that it can't just be `abc` or `alembic` or whatever?

Reply:
We need this family just for the extractor, the actual family will be `pointcache` or `model`. I think it makes it a bit less ambiguous!

Reply:
maybe instead of being extractor here, it can be somehow the collector similar to the gltf collector in maya?
https://github.com/ynput/OpenPype/blob/d1e1e591f2b0f6fb77cf1d119c9816a705e86723/openpype/hosts/maya/plugins/publish/collect_gltf.py#L1-L17

Reply:
A collector cannot be set optional though!

Reply:
As far as I know from other host, i dont think it can't be set optional.
![image](https://github.com/ynput/OpenPype/assets/64118225/e3344335-0555-4e9f-871a-6362fb83118d)
https://github.com/ynput/OpenPype/blob/135f2a9e5741d7623417bd944dbc066af56fd9f4/openpype/hosts/maya/plugins/publish/collect_fbx_animation.py#L7-L15

Reply:
It can be optional with the new publisher. But in the old publisher the collector needs to be turned off by the OP settings. 
I can see why it sets up like this. But I am thinking if it's possible to deal with the extractors(whether being optional or not)  in different ways. Like the pesudo code below:(I dont know if it's possible as it is with different families)
```
class ExtractAbc(publish.Extractor):
    """Extract camera as ABC."""

    label = "Extract Pointcache (ABC)"
    hosts = ["blender"]
    families = ["pointcache"]

    def process(self, instance):
        # Define extract output file path
        "your code here...."


class ExtractAbcModel(ExtractAbc):
    label = "Extract Model (ABC)"
    hosts = ["blender"]
    families = ["model"]
    optional = True

```

Reply:
@moonyuet this actually works very well and I like it a lot! I'll change it to this :) Thanks!